RFC/src/RFC-0130_Mining.md

```diff
@@ -1 +1,93 @@
+
+## Related RFCs
+
+* [RFC-0100: Base layer](RFC-0100_BaseLayer.md)
+
+## Description
+
+### Assumptions
+- That the Tari blockchain will be merged mined with Monero 
+
+### Abstract
+
+The process of mining on the Tari base layer is responsible for confirming and adding valid transactions to the Tari blockchain and distributing transactions on the Tari base layer network. This task is achieved by validating transactions and by performing Nakamoto consensus through Proof-of-Work. New blocks on the Tari blockchain will be merged mined with Monero by linking Tari blocks and Monero blocks.  
+
+### Full-node mining on Tari base layer
+
+The document is divided into two parts. First an overview will be provided describing the Tari merged mining process, then a descriptive list of the primary functionality required by the Tari full-node miner will be proposed.
+
+
+####  Overview of Tari merged mining process
+
+Valid transactions that need to be included on the Tari blockchain should be propagated by the mining full-nodes on the Tari base layer network. A Tari mining full-node should retrieve transactions from its mempool and assemble a new Tari block by bundling transactions together. It should ensure that the transactions that are included in the new Tari block are valid and that rules such as signatures and timelocks are enforced before they are included in a new block.
+
+As Tari is merged mined with Monero, both a Tari and a Monero block need to be created and linked by including some Tari block information in the Monero block, and some Monero block information in the Tari block. First, a new Tari block is created and then the block header hash of the new Tari block should be included in the coinbase transaction of the new Monero block. Once the block construction is complete, the mining full-node should perform the CryptoNight Proof-of-Work (PoW) algorithm on the Monero block that includes the information of the new Tari block. 
+
+The solution to the PoW problem could be solved at the difficulty of either the Tari and/or Monero blockchain. If a solution has been found that meets the minimum difficulty requirements of the Monero blockchain, then the new Tari block should be updated by including the solved Monero block’s information (block header hash, Merkel tree branch, and hash of the coinbase transaction) into the PoW summary section of the Tari block header. If a solution was found that meet the Tari or Monero blockchain difficulty then the new Tari block can be added to the Tari blockchain. If the solution met the difficulty requirements of the Monero blockchain then the new Monero block can also be added to the Monero blockchain. If the PoW solution was sufficient to meet the difficult level of both the Tari and Monero blockchains then the individual blocks for each cryptocurrency can be added to their respective blockchains. The Tari block times is (hard fork) flexible and can be less than, equal or greater than the Monero block times.
```

Comment:
Comma needed here (^): "_If a solution was found that meet the Tari or Monero blockchain difficulty^ then the_"

Reply:
Suggested change implemented